frontend/src/metabase/models/components/ModelDetailPage/ModelActionDetails/ModelActionDetails.tsx

```diff
@@ -218,7 +218,7 @@ function NoActionsState({
 
 function mostRecentFirst(action: WritebackAction) {
   const createdAt = parseTimestamp(action["created_at"]);
-  return -createdAt.unix();
+  return -createdAt.valueOf();
```

Comment:
This is the fix. [`moment.unix()`](https://momentjs.com/docs/#/displaying/unix-timestamp/) would return the Unix timestamp in seconds, but [`moment.valueOf()`](https://momentjs.com/docs/#/displaying/unix-timestamp-milliseconds/) would return the Unix timestamp in milliseconds.